pwa/app/routes/event.$eventKey.tsx

```diff
@@ -416,3 +428,17 @@ function TeamsTab({
     </div>
   );
 }
+
+function InsightsTab({
+  coprs,
+  colors,
+}: {
+  coprs: EventCopRs;
+  colors: EventColors;
+}) {
```

Comment:
This might be a good time to experiment with using react query for client-side fetching "non-critical" data (especially since this is on a different tab) so we don't block the page load waiting for colors.